iambic/config/wizard.py

```diff
@@ -418,6 +387,43 @@ def cf_role_arn(self):
 
         return self._cf_role_arn
 
+    @property
+    def has_aws_account_or_organizations(self) -> bool:
+        return self.config.aws and (
+            self.config.aws.accounts or self.config.aws.organizations
+        )
+
+    @property
+    def secrets_message(self) -> tuple[str, bool]:
+        """Returns if secrets are configured"""
+        secret_in_config = bool(self.config.extends)
+
+        if (
+            secret_in_config
+            and self.config.extends[0].key == ExtendsConfigKey.AWS_SECRETS_MANAGER
+        ):
+            secret_question_text = (
+                "This requires the ability to update the AWS Secrets Manager secret."
+            )
+        elif (
+            secret_in_config
+            and self.config.extends[0].key == ExtendsConfigKey.LOCAL_FILE
+        ):
+            secret_question_text = (
+                "This requires permissions to update secrets to you secrets file."
```

Comment:
This requires permissions to update secrets to you secrets file.

Should this be: This requires permissions to update secrets **in your** secrets file. ?